### ISSUE TITLE
[UI] - Update Sidebar of Meshery According to New Figma Design

### ISSUE THREAD

**Issue by adithyaakrishna** [meshery/meshery#5042]
#### Current Behavior

The sidebar in Meshery UI looks kinda like this now, which needs to be updated according to the new design and prototype in figma
![image](https://user-images.githubusercontent.com/23498248/152600103-de08bc29-138a-4da4-9201-a0f2edede55c.png)

#### Desired Behavior

**Proposed Non-Collapsed Design**

![image](https://user-images.githubusercontent.com/23498248/152600547-38c38676-ad67-4a43-acbb-6a0cbb811cb6.png)

**Proposed Collapsed Design**

![image](https://user-images.githubusercontent.com/23498248/152600573-17233719-24c3-414c-9532-87deda219ce5.png)

**Proposed Hover States For Menu Items:**

![image](https://user-images.githubusercontent.com/23498248/152601063-70e919cb-2a7e-482b-b48e-1c17542a9ee9.png)

**Proposed Hover States For Sub-Menu Items:**

![image](https://user-images.githubusercontent.com/23498248/152600922-4d23a8bb-3cd8-4d14-821c-2ec4ee51bc40.png)

You can refer the Complete Prototype below to get an idea of what is expected or is to be improved :)

#### Mockups

1. **Non-Collapsed Figma Link:** https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=4275%3A23971

2. **Collapsed Figma Link:** https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=4313%3A28988

3. **Complete Prototype:** https://www.figma.com/proto/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=4275%3A23971&scaling=scale-down&page-id=347%3A6794&starting-point-node-id=4275%3A23971&show-proto-sidebar=1

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by leecalcote** [comment]
@YashKamboj did your implementation of this land, yet?

**Comment by leecalcote** [comment]
The hover state here isn't quite right. The chevron/arrow should be right aligned ~1rem away from the edge of the sidebar.

**Comment by leecalcote** [comment]
This design looks great and has incorporated all feedback. We're good to go on implementation.

**Comment by asubedy** [comment]
I can work on this

**Comment by YashKamboj** [comment]
@leecalcote Yeah!! was done days ago can check #4926 , i can make the new changes of the hover and the new dropdown in that  pr only
